### ISSUE TITLE
AttributeError when importing Keras

### ISSUE THREAD

**Issue by disssid** [microsoft/MCW-ML-Ops#41]
Azure - Azure trial subscription

Issue : 

```
AttributeError  

Traceback (most recent call last)

<ipython-input-3-9c78a7be919d> in <module>()
      5 
      6 import cv2
----> 7 import keras
      8 import numpy as np
      9 import matplotlib.pyplot as plt

8 frames

/usr/local/lib/python3.7/dist-packages/keras/initializers/__init__.py in populate_deserializable_objects()
     47 
     48   LOCAL.ALL_OBJECTS = {}
---> 49   LOCAL.GENERATED_WITH_V2 = tf.__internal__.tf2.enabled()
     50 
     51   # Compatibility aliases (need to exist in both V1 and V2).

AttributeError: module 'tensorflow.compat.v2.__internal__' has no attribute '__internal__'
```

#### File: Deep learning.ipynb

```
from azureml.widgets import RunDetails
from azureml.train.dnn import TensorFlow

import keras
import tensorflow
from keras.models import load_model
```

Same issue when converting the Keras model to ONNX

#### How I fixed it:
Referring to the answer provided here - https://stackoverflow.com/a/67787883

For the import issues, changed the imports to the following
```
import tensorflow
from tensorflow import keras
from tensorflow.keras.models import load_model
from tensorflow.keras.preprocessing.text import Tokenizer
from tensorflow.keras.preprocessing.sequence import pad_sequences
```

Regarding the Keras model to ONNX conversion:
I installed the latest version of onnxmltools, keras2onnx, onnxruntime, tf2onnx using
`!pip install onnxmltools keras2onnx onnxruntime tf2onnx`

This fixed the issue and helped me complete the workshop.

**Comment by shirolkar** [comment]
@DawnmarieDesJardins @timahenning I have sent a PR for this fix. Thanks. Jitendra

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #42
